tests/codebuild/run_integtest.sh

```diff
@@ -6,6 +6,7 @@
 # crd_namespace is currently hardcoded here for cleanup. Should be an env variable set in the pipeline and .env file
 
 source tests/codebuild/common.sh
+default_operator_namespace="sagemaker-k8s-operator-system"
 crd_namespace=${NAMESPACE_OVERRIDE:-"test-namespace"}
```

Comment:
I just realised that this variable name doesn't make much sense. The CRDs are global, so there is no "CRD namespace". I understand this was for the namespace-scope tests, so I should have caught it in the previous PR.

Reply:
This was a todo in my previous PR. You used the variable crd_namespace in your branch, which i picked and now its everywhere. Needs a massive cleanup.

Reply:
Oh oops. I was going to do a big cleanup in my next refactor, so that's fine I can take care of it.